packages/opentelemetry-cloud-trace-exporter/src/trace.ts

```diff
@@ -95,49 +111,44 @@ export class TraceExporter implements SpanExporter {
    * service.
    * @param spans
    */
-  private _batchWriteSpans(spans: SpansWithCredentials) {
+  private _batchWriteSpans(spans: NamedSpans) {
     this._logger.debug('Google Cloud Trace batch writing traces');
-
     return new Promise((resolve, reject) => {
-      // @todo Consider to use gRPC call (BatchWriteSpansRequest) for sending
-      // data to backend :
-      // https://cloud.google.com/trace/docs/reference/v2/rpc/google.devtools.
-      // cloudtrace.v2#google.devtools.cloudtrace.v2.TraceService
-      TraceExporter._cloudTrace.projects.traces.batchWrite(
-        spans,
-        (err: Error | null) => {
-          if (err) {
-            err.message = `batchWriteSpans error: ${err.message}`;
-            this._logger.error(err.message);
-            reject(err);
-          } else {
-            const successMsg = 'batchWriteSpans successfully';
-            this._logger.debug(successMsg);
-            resolve(successMsg);
-          }
+      if (!this._traceServiceClient) {
+        return reject(new Error('channel not yet opened'));
+      }
+      this._traceServiceClient.BatchWriteSpans(spans, (err: Error) => {
+        if (err) {
+          err.message = `batchWriteSpans error: ${err.message}`;
+          this._logger.error(err.message);
+          reject(err);
+        } else {
+          const successMsg = 'batchWriteSpans successfully';
+          this._logger.debug(successMsg);
+          resolve(successMsg);
         }
-      );
+      });
     });
   }
 
   /**
-   * Gets the Google Application Credentials from the environment variables,
-   * authenticates the client and calls a method to send the spans data.
-   * @param spans The spans to export
+   * Initializes the cloudtrace rpc client
    */
-  private async _authorize(
-    spans: Span[]
-  ): Promise<SpansWithCredentials | null> {
-    try {
-      return {
-        name: `projects/${this._projectId}`,
-        resource: { spans },
-        auth: await this._auth.getClient(),
-      };
-    } catch (err) {
-      err.message = `authorize error: ${err.message}`;
-      this._logger.error(err.message);
-      return null;
-    }
+  private _init(creds: Compute | JWT | UserRefreshClient): void {
+    this._logger.debug('Google Cloud Trace initializing rpc client');
+    const pacakageDefinition = protoloader.loadSync(
+      protofiles.getProtoPath('devtools', 'cloudtrace', 'v2', 'tracing.proto'),
+      {
+        includeDirs: [protofiles.getProtoPath('..')],
+      }
+    );
+    const { google }: any = grpc.loadPackageDefinition(pacakageDefinition);
```

Comment:
The `any` typing here is causing the linter to fail. I'm not exactly sure how I can avoid using `any` since the proto-loader doesn't generate typings